src/index.js

```diff
@@ -0,0 +1,34 @@
+import UI from './components/ui';
+import Weather from './components/weather';
+import Storage from './components/storage';
+import './css/style.css';
+
+const storage = new Storage();
+const weatherLocation = storage.getLocationData();
+const weather = new Weather(weatherLocation.city);
+
+const ui = new UI();
+
+
+function getWeather() {
+  weather.getWeather()
+    .then(results => {
+      ui.paint(results);
+    })
+    .catch(err => err);
+}
```

Comment:
Kindly use the ES6 arrow functions. This will ensure that your code is consistent in following the right Es6 syntax.  ⬇️ 
![es6_requirement](https://user-images.githubusercontent.com/32361963/109619567-fbaa4080-7b49-11eb-9fea-b3bcd3f3ee2f.png)